### DISCUSSION TITLE
RuntimeException: The source file "/var/www/wintercms/themes/responsiv-flat/assets/css/theme.css" does not exist

### DISCUSSION THREAD

**Discussion by GoSpursGoNL** [wintercms/winter#299] [Q&A] [answered]
Having used OctoberCMS in the past, I did a fresh install of WinterCMS via the web installer. The website is working using the standard demo theme. Then I installed the responsiv-flat theme via the backend, which goes fine. I activate the theme via the 'frontend theme' setting.
Then, when I visit the site, I only see a blank page.

In the backend -> event log I see error messages appearing afterwards:
Symfony\Component\Debug\Exception\FatalErrorException: Uncaught RuntimeException: The source file "[...]/var/www/wintercms/themes/responsiv-flat/assets/css/theme.css" does not exist. in /var/www/wintercms/vendor/winter/storm/src/Assetic/Asset/FileAsset.php:71

and:
RuntimeException: The source file "/var/www/wintercms/themes/responsiv-flat/assets/css/theme.css" does not exist. in /var/www/wintercms/vendor/winter/storm/src/Assetic/Asset/FileAsset.php:71

Am I doing something wrong? When I check my filesystem there is no css folder in the assets folder of the theme. The folders in there are: images, javascript, less, vendor.

**Comment by LukeTowers** [comment, marked as answer]
@GoSpursGoNL does your layout file include the changes from https://github.com/responsiv/flat-theme/commit/4a4a022531ee1f18be2a83c455b6b701ecc0bef6?

**Comment by GoSpursGoNL** [comment]
Thanks a lot! Indeed my layout file (coming from OctoberCMS installation) didn't include those changes and also had outdated references related to stylesheet, jquery and bootstap-related filenames. Updated, and now the website is working. I'll continue with my migration :)
